corebehrt/functional/utils.py

```diff
@@ -106,9 +144,91 @@ def remove_missing_timestamps(df: pd.DataFrame) -> pd.DataFrame:
     return df[df.TIMESTAMP.notna()]
 
 
-def get_first_event_by_pid(df: pd.DataFrame):
+def get_first_event_by_pid(df: pd.DataFrame, timestamp_column: str) -> pd.DataFrame:
```

Comment:
Assume `abspos` like other functions do

Reply:
Yes, I think it would be a good idea to have these constants in a script and import them